akka-docs/src/main/paradox/typed/persistence-durable-state.md

```diff
@@ -0,0 +1,391 @@
+---
+project.description: Durable State with Akka Persistence enables actors to persist its state for recovery on failure or when migrated within a cluster.
+---
+# Durable State
+
+## Module info
+
+To use Akka Persistence, add the module to your project:
+
+@@dependency[sbt,Maven,Gradle] {
+  bomGroup=com.typesafe.akka bomArtifact=akka-bom_$scala.binary.version$ bomVersionSymbols=AkkaVersion
+  symbol1=AkkaVersion
+  value1="$akka.version$"
+  group=com.typesafe.akka
+  artifact=akka-persistence-typed_$scala.binary.version$
+  version=AkkaVersion
+  group2=com.typesafe.akka
+  artifact2=akka-persistence-testkit_$scala.binary.version$
+  version2=AkkaVersion
+  scope2=test
+}
+
+You also have to select journal plugin and optionally snapshot store plugin, see 
+@ref:[Persistence Plugins](../persistence-plugins.md).
+
+@@project-info{ projectId="akka-persistence-typed" }
+
+## Introduction
+
+This model of Akka Persistence enables a stateful actor / entity to store the full state afer processing each command instead of using event sourcing. This reduces the conceptual complexity and can be a handy tool for simple use cases. Very much like a CRUD based operation, the API is conceptually simple - a function from current state and incoming command to the next state which replaces the current state in the database. 
+
+```
+(State, Command) => State
+```
+
+The current state is always stored in the database. Akka Persistence would read that state and store it in memory. After processing of the command is finished, the new state will be stored in the database. The processing of the next command will not start until the state has been successfully stored in the database.
```

Comment:
Maybe make it clear that the storage replaces with every change (no audit or history kept)

Reply:
done ..